### PR TITLE
Create kb-modifying-exported-xlsx-gridview-spreadsheet.md

### PR DESCRIPTION
**Pull request by LanceMcCarthy**

### CODE REVIEW

File: knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Modifying GridView Exported XLSX Spreadsheet
+description: How to make cusomt changes to exported spreadsheet document when exporting GridView to Excel xlsx.
```
Comment (PMcDonough11) [inline knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: - Instead of "How to", consider starting with a verb.  
- *custom

Comment (LanceMcCarthy) [reply knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: Also caught a misspelling there (custom)

File: knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    using (var output = dialog.OpenFile())
+    {
+        new XlsxFormatProvider().Export(workbook, output);
+    }
+}
+```
+
+## Resources
+
+For more settings and to gain an better understanding of what the  linker does, visit the following documentation article:
```
Comment (PMcDonough11) [inline knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: remove space before "linker"

File: knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+title: Modifying GridView Exported XLSX Spreadsheet
+description: How to make cusomt changes to exported spreadsheet document when exporting GridView to Excel xlsx.
+type: how-to
+page_title: Customize Excel XLSX Spreadhseet After Exporting From GridView
```
Comment (PMcDonough11) [inline knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: *Spreadsheet

File: knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md (hunk 2 above)
Comment (LanceMcCarthy) [reply knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: The entire line was invalid and a copy/pasta fail on my part.

**Comment by LanceMcCarthy** [comment]
Looping in @martinivanoff for WPF team's policy/process toward merging new KBs.

File: knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: Modifying GridView Exported XLSX Spreadsheet
+description: Apply custom changes to the spreadsheet document when exporting GridView to Excel xlsx.
+type: how-to
+page_title: Customize Excel XLSX Spreadsheet When Exporting From RadGridView
+slug: kb-modifying-gridview-exported-xlsx-spreadsheet
+position:
```
[the remark this answers is not in the document]
Comment (LanceMcCarthy) [reply knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md]: I'm guessing you already added 0 for me? Thanks!

**Pull request merged**